### DISCUSSION TITLE
How can one replace existing WMS/WFS service with  pygeoapi.

### DISCUSSION THREAD

**Discussion by 1ly1mk** [geopython/pygeoapi#1615] [General]
We publish our data(Raster+Vector) as OWS services(WMS/WMTS/WFS) and consume through Mapbox. I have gone through OGC Map & Tile API which are possible alternatives but haven't tested yet. What is an appropriate solution here? 
@tomkralidis @krishnaglodha

**Comment by whatnick** [comment]
You can go through the samples here to bridge the existing services with PygeoAPI and support the newer standards.

**Comment by matrss** [comment]
Is it possible to go the other way, i.e. have a pygeoapi-based server and access its data e.g. via OGC WMS? Does some kind of proxy for that exist? I am looking at using pygeoapi for a new project, but some client applications only support the older protocols.

**Reply by tomkralidis** [reply]
@matrss this is not possible, given pygeoapi's focus is on providing OGC API services.
